packages/block-library/src/style.scss

```diff
@@ -217,28 +217,30 @@
 
 
 // Font sizes.
+:root {
```

Comment:
Ideally, this could use the CSS variables as well, can do in a separate PR. cc @jorgefilipecosta as you were thinking about this.

Reply:
Follow-up at https://github.com/WordPress/gutenberg/pull/21429